examples/js/loaders/sea3d/SEA3DLoader.js

```diff
@@ -1695,7 +1695,7 @@ THREE.SEA3D.prototype.updateTransform = function ( obj3d, sea ) {
 
 	// ignore rotation scale
 
-	obj3d.rotation.setFromRotationMatrix( THREE.SEA3D.identityMatrixScale( mtx ) );
+	obj3d.quaternion.setFromRotationMatrix( THREE.SEA3D.identityMatrixScale( mtx ) );
```

Comment:
/ping @Sunag Should be OK as long as the argument is a pure rotation matrix.

Reply:
/ping @sunag opinion??

Reply:
> /ping @sunag Should be OK as long as the argument is a pure rotation matrix.

Yes, yes! Looks good